distutils/unixccompiler.py

```diff
@@ -144,6 +144,9 @@ class UnixCCompiler(CCompiler):
     xcode_stub_lib_format = dylib_lib_format
     if sys.platform == "cygwin":
         exe_extension = ".exe"
+        shared_lib_extension = ".dll.a"
+        dylib_lib_extension = ".dll"
+        dylib_lib_format = "cyg%s%s"
```

Comment:
This looks like an artifact of something merged from another review... possibly brought about by my force-push to main in #263. I'll want to double-check the diff and tree.

Reply:
I couldn't tell why, but there were three commits near the top that seem to have been rebased. I'm surprised they show up in the diff. In any case, after rebasing the 5 commits onto the current main, these unrelated changes disappear.